scripts/create_metadata

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+
+if [ -z "${DEV_TAGS+x}" ]; then
+    jq --null-input --arg tags "${TAGS}" \
+    '{
+        tags: $tags | split("\n")[:-1]
+    }' >> docker_tag_list_"${TARGET}".json
+
+    echo $PWD
+else
+   jq --null-input --arg tags "${TAGS}" \
+    --arg dev_tags "${DEV_TAGS}" \
+    '{
+        tags: $tags | split("\n")[:-1],
+        dev_tags: $dev_tags | split("\n")[:-1]
+    }' >> docker_tag_list_"${TARGET}".json
+
+    echo $PWD
+fi
```

Comment:
Would it be possible to always include the `dev_tags` even when it's empty? I'm just thinking that an explicitly empty array seems clearer/easier to parse than a missing field, and it could also make the script shorter.

(Some parsers consider it an error if you try to access a missing field, whereas some just give you `null`. Not sure how `jq` handles it, but in the future we might use other parsers, so best to keep all the fields present to make it easy to switch later if we want to.)

Reply:
Makes sense! I've updated the script to always include `dev_tags`